### PR TITLE
Updating Marlowe market contract templates.

### PR DESCRIPTION
**Pull request by merivale**
This updates the web-common-marlowe Market with the new contract templates by @palas available here: https://gist.github.com/palas.

### CODE REVIEW

File: web-common-marlowe/src/Marlowe/Market/Contract1.purs [hunk 1]
```diff
@@ -18,185 +18,123 @@ contractTemplate = { metaData, extendedContract }
...
+        , "Buyer" /\ "The party that wants to buy the item. Payment is made to the seller if they acknowledge receiving the item. "
+        , "Seller" /\ "The party that wants to sell the item. They receive the payment if the exchange is uneventful."
         ]
   , slotParameterDescriptions:
       fromFoldable
-        [ "aliceTimeout" /\ "about the aliceTimeout"
-        , "arbitrageTimeout" /\ "about the arbitrageTimeout"
-        , "bobTimeout" /\ "about the bobTimeout"
-        , "depositSlot" /\ "about the depositSlot"
+        [ "Buyer's deposit timeout" /\ "Deadline by which the \"Buyer\" must deposit the selling  \"Price\" in the contract."
```
Comment (hrajchert) [inline web-common-marlowe/src/Marlowe/Market/Contract1.purs:31]: @palas , in a future PR we should remove all Slot Params, right?

File: web-common-marlowe/src/Marlowe/Market/Contract2.purs [hunk 2]
```diff
@@ -18,184 +18,138 @@ contractTemplate = { metaData, extendedContract }
...
+                                            ( Choice
+                                                ( ChoiceId
+                                                    "Dispute problem"
+                                                    (Role "Seller")
+                                                )
+                                                [ Bound zero zero ]
+                                            )
+                                            ( Pay
+                                                (Role "Seller")
+                                                (Party (PK "0000000000000000000000000000000000000000000000000000000000000000"))
```
Comment (hrajchert) [inline web-common-marlowe/src/Marlowe/Market/Contract2.purs:119]: what is this PK? Shouldn't we use a role?

File: web-common-marlowe/src/Marlowe/Market/Contract5.purs [hunk 3]
```diff
@@ -5,197 +5,81 @@ module Marlowe.Market.Contract5
...
-                                Close
-                            )
-                        ]
-                        (SlotParam "arbitrageTimeout")
-                        Close
-                    )
+                ( Deposit
+                    (Role "Dollar provider")
+                    (Role "Dollar provider")
+                    (Token "85bb65" "dollar")
```
Comment (hrajchert) [inline web-common-marlowe/src/Marlowe/Market/Contract5.purs:60]: I thought the first value of a Token construct was the symbol we used to represent the token. For example $... When we show the currency, what should we show in this case? 

Should we have a Map of common tokens to symbols and if it's not found use the second argument as a postfix?

In this case if we didn't have `"85bb65" -> "$"` we could say `1500 dollar`

File: web-common-marlowe/src/Marlowe/Market/Contract5.purs [hunk 4]
```diff
@@ -5,197 +5,81 @@ module Marlowe.Market.Contract5
...
-                            "choice"
-                            (Role "carol")
-                        )
-                        [ Bound zero zero ]
+                ( Pay
+                    (Role "Ada provider")
+                    (Party (Role "Dollar provider"))
+                    (Token "" "")
+                    ( MulValue
+                        (Constant $ BigInteger.fromInt 1000000)
```
Comment (hrajchert) [inline web-common-marlowe/src/Marlowe/Market/Contract5.purs:68]: Eventually this should be an oracle, right? We probably need a way to display this to the users somehow 🤔

**Review by hrajchert — APPROVED** [review]
LGTM but I added some comments

File: web-common-marlowe/src/Marlowe/Market/Contract1.purs (hunk 1 above)
Comment (palas) [reply web-common-marlowe/src/Marlowe/Market/Contract1.purs:31]: We removed some of them, but when they are few they don't look bad. It is up to you, we can have them in the playground for example

File: web-common-marlowe/src/Marlowe/Market/Contract2.purs (hunk 2 above)
Comment (palas) [reply web-common-marlowe/src/Marlowe/Market/Contract2.purs:119]: It is to burn money, it is not a participant, it just receives the payment

File: web-common-marlowe/src/Marlowe/Market/Contract5.purs (hunk 3 above)
Comment (palas) [reply web-common-marlowe/src/Marlowe/Market/Contract5.purs:60]: I think it is the hash of the token and the readable symbol is the second field, we can show the hash in a hint maybe. @nau knows better

**Review by palas — APPROVED** [review]
👍

File: web-common-marlowe/src/Marlowe/Market/Contract5.purs (hunk 4 above)
Comment (palas) [reply web-common-marlowe/src/Marlowe/Market/Contract5.purs:68]: Maybe, yes, I am not sure how oracles are going to work on the prototype, nor tokens, we need to discuss this later and we can update the examples then

**Pull request merged**